python/ray/serve/api.py

```diff
@@ -935,3 +961,8 @@ async def __call__(self, val: int) -> int:
         ServeUsageTag.SERVE_GET_DEPLOYMENT_HANDLE_API_USED.record("1")
 
     return client.get_handle(deployment_name, app_name, check_exists=_check_exists)
+
+
+@PublicAPI
+def get_tracing_context() -> Optional[Dict]:
```

Comment:
Was this used anywhere??

Reply:
Oops this is not supposed to be here, removed.